platform/src/main/resources/assets-prod.json

```diff
@@ -62,6 +62,7 @@
                     "stellar:USDC:GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN"
                 ],
                 "country_codes": ["USA"],
+                "decimals": 4,
```

Comment:
Reading from assets-prod is legacy code from MGI. We need to externalize the assets information (not in the code/resource).
Issue [#128](https://github.com/stellar/java-stellar-anchor-sdk/issues/128) is created.